Widgets/widget-IndicatorActivity.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "indicators-activity",
+  "version": -1,
+  "name": "Indicators Activity",
+  "dataType": "indicators",
+  "widgetType": "line",
+  "size": 1000,
```

Comment:
Why the limit. We return just groups.

Reply:
Limits to number of points loaded

Reply:
Than we should add it to all line charts